db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_07_06_090950) do
+ActiveRecord::Schema.define(version: 2022_07_07_145032) do
 
-  create_table "activity_logs", force: :cascade do |t|
+  create_table "activity_logs", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
schema.rb needs cleaning up (by running migrations using sqlite3, and then fixing some bigints that get created), but I can do that after being merged

Reply:
Fixed.